### PR TITLE
Tenant vos

### PR DESCRIPTION
**Pull request by bodnara**
* added page for managing member organizations of hierarchical org.
* added dialog to add/remove member org.
* added new icon for hierarchical org.
* member/parent vos are included  on org. detail

### CODE REVIEW

**Review by xkureck — CHANGES_REQUESTED** [review]
Hello, so fix the space between the names of members vo, here is something that can help  you

https://www.tutorialrepublic.com/faq/how-to-remove-the-space-between-inline-block-elements-in-css.php

Also it would be nice if you used the universal remove dialog

**Comment by bodnara** [comment]
Universal remove dialog was utilized.
Regarding the spacing, I couldn't find a simple solution and realized that this whole section is not very useful and should probably be removed anyway.

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-member-organizations/vo-settings-member-organizations.component.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      theme: 'vo-theme',
+    };
+
+    this.dialog
+      .open(UniversalRemoveItemsDialogComponent, config)
+      .afterClosed()
+      .subscribe((result) => {
+        if (result) {
+          this.removeMemberVos();
+          this.reloadDetailService.reloadEntityDetail();
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-member-organizations/vo-settings-member-organizations.component.ts:100]: reload entity detail doesnt wait for remove member vos to end so the entity is reloaded with old data

**Review by xkureck — CHANGES_REQUESTED** [review]
btw the stakeholders also wanted to have "tenant" icon in the list of all vos in the select vo table
image

File: apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-detail-page.component.html [hunk 2]
```diff
@@ -33,7 +33,15 @@
         </div>
         <span class="mt-1 entity-info">
           {{'VO_DETAIL.ENTITY' | translate}}, {{'VO_DETAIL.SHORT_NAME' | translate}}:
-          {{vo.shortName}}
+          {{vo.shortName}},
```
Comment (xkureck) [inline apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-detail-page.component.html:36]:
```suggestion
          {{vo.shortName}}<span *ngIf="enrichedVo.memberVos.length !== 0 || enrichedVo.parentVos.length !== 0">, </span>
```

we don't want the "," here if there is no continuing

**Review by xkureck — CHANGES_REQUESTED** [review]
this is really the last thing

**Review by xkureck — APPROVED** [review]

**Pull request merged**